ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -806,6 +806,10 @@ ethernet_interfaces:
       trust: < dscp | cos >
       dscp: < dscp-value >
       cos: < cos-value >
+    priority_flow_control: < on | off >
```

Comment:
I would suggest to adjust the data model a bit to make it more consistent with what we have elsewhere:
```yaml
priority_flow_control:
  enabled: < true | false >
  priorities:
    - priority: < 0-7 >
      no_drop: < true | false >
```

Reply:
updated

Reply:
The datamodel is still split into multiple main keys. Please take a look again at my suggested model above, and let me know if we need to discuss it further.